kuma/static/js/interactive.js

```diff
@@ -7,6 +7,84 @@
+            var currentEntry = perfEntries[i];
+            if (
+                currentEntry.initiatorType &&
+                currentEntry.initiatorType === 'iframe' &&
+                currentEntry.name.indexOf(
+                    'interactive-examples.mdn.mozilla.net'
+                ) > -1
+            ) {
+                return perfEntries[i];
+            }
+        }
+    }
+
+    /* Ensure there is an iframe present and that performance is
+       supported before gathering performance metric */
+    if (iframe && performance !== undefined) {
+        document.addEventListener('readystatechange', function(event) {
+            if (event.target.readyState === 'complete') {
+                var interactiveExamplesPerfEntry = {};
+                var perfEntries = performance.getEntriesByType('resource');
+                var mainFetchStart = 0;
+                var iframeFetchStart = 0;
+                var iframeFetchStartSinceUnixEpoch = 0;
+
+                if (perfEntries === undefined || perfEntries.length <= 0) {
+                    console.info('No performance entries was returned');
+                    return;
+                }
+
+                interactiveExamplesPerfEntry = getInteractiveExamplesPerfEntry(
+                    perfEntries
+                );
+
+                mainFetchStart = performance.timing.fetchStart;
+                iframeFetchStart = Math.round(
+                    interactiveExamplesPerfEntry.fetchStart
+                );
+                iframeFetchStartSinceUnixEpoch =
+                    mainFetchStart + iframeFetchStart;
+
+                // var iframeLoadTime = interactiveExamplesPerfEntry.responseEnd - interactiveExamplesPerfEntry.startTime;
+                // ga('send', {
+                //     'eventCategory' : 'Interactive Examples',
+                //     'eventAction'   : 'iFrame load time',
+                //     'eventLabel'    : 'Performance Events',
+                //     'eventValue'    : (iframeLoadTime / 1000),
+                //     'hitType': 'timing'
+                // });
+                console.log(
+                    (interactiveExamplesPerfEntry.responseEnd -
+                        interactiveExamplesPerfEntry.startTime) /
+                        1000
+                );
+
+                // var timeToIframeFetchStart = new Date(iframeFetchStartSinceUnixEpoch).getSeconds() -
+                //    new Date(mainFetchStart).getSeconds()
+                // ga('send', {
+                //     'eventCategory' : 'Interactive Examples',
+                //     'eventAction'   : 'Time to iframe fetch start',
+                //     'eventLabel'    : 'Performance Events',
+                //     'eventValue'    : timeToIframeFetchStart,
+                //     'hitType': 'timing'
+                // });
+                console.log(
+                    'Time in seconds between fetch start of main page, and fetch start of iframe',
```

Comment:
Same thing here. Milliseconds would be nice for more precision. So something like

```js
new Date(iframeFetchStartSinceUnixEpoch) - new Date(mainFetchStart)
```